modules/ROOT/pages/apikit-4-soap-error-handling-reference.adoc

```diff
@@ -0,0 +1,29 @@
+= APIkit for SOAP Error Handling Reference
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+APIkit offers an error handling mechanism for your application process. This article lists the typed errors for APIkit for SOAP. 
+
+== APIkit for SOAP Error Types
+
+APIkit for SOAP defines two error types:
+
+[%header]
+|===
+|Error Type |Description
+
+|APIKIT-SOAP:ROUTER
+|Routing exception
+
+|APIKIT-SOAP:FAULT
+|Fault exception
+|===
+
+== APIkit for SOAP Error Handling
+
+APIkit for SOAP scaffolder does not generate error-handling code.
```

Comment:
The verb here should be parallel to the verb in first sentene of APIkit for SOAP Error Types.

Reply:
OK! Thanks!